app/models/task.rb

```diff
@@ -26,7 +26,6 @@ class Task < ApplicationRecord
     order(Arel.sql("CASE aasm_state WHEN 'completed' THEN 1 ELSE 2 END"), created_at: :desc)
```

Comment:
Not that there's likely to be a field called `aasm_state` on `tasks` because `status` already *is* an `aasm_state` field by another name, but this probably ought to be qualified `submissions.aasm_state`, because if it isn't, and we join a scope to A.N. Other table with an `aasm_state` in it we'll get an ambiguous field error.